src/Entities/Player.cpp

```diff
@@ -1942,6 +1942,26 @@ void cPlayer::TossEquippedItem(char a_Amount)
 
 
 
+void cPlayer::ReplaceOneEquippedItemTossRest(const cItem & a_Item)
+{
+	auto Placed = GetInventory().ReplaceOneEquippedItem(a_Item);
```

Comment:
In order for the code to be immediately readable without needing to check the type that `ReplaceOneEquippedItem` returns, the variable should be named `NumPlaced` or similar, to indicate that it contains amount, rather than a `cItem`.